### ISSUE TITLE
"Archived" badge overlapping.

### ISSUE THREAD

**Issue by Akarshit** [reactioncommerce/reaction#2961]
### Expected behavior
The badges should not overlap.
### Actual Behavior
Archived badge is overlapping with the edit badge.
<img width="514" alt="screen shot 2017-09-27 at 6 46 16 pm" src="https://user-images.githubusercontent.com/7762605/30915376-43b22486-a3b4-11e7-9be7-04f868b340a4.png">

### Steps to Reproduce the Behavior
Archive a product with sub-variant.
### Versions
```
Node: 8.5.0
NPM: 5.3.0
Meteor Node: 4.8.4
Meteor NPM: 4.6.1
Reaction CLI: 0.12.0
Reaction: 1.5.0
Reaction branch: marketplace
```

### Solution
Instead of using "Achived" we should use a icon that means "Archived".

**Comment by kieckhafer** [comment]
I'd re-check this issue before you do any work on it, because we removed the visibility button from the variants, so there is more room there / this might not be occurring anymore.

**Comment by foladipo** [comment]
@kieckhafer Thanks. I did notice that there was more room on the button because the visibility icon had been removed. But then the overlapping still happens at ~350px screen width.

**Comment by foladipo** [comment]
Resolved by #3198.
